vscode/microsoft-kiota/src/steps.ts

```diff
@@ -72,28 +72,17 @@ export async function searchSteps(searchCallBack: (searchQuery: string) => Thena
             validate: validateIsNotEmpty,
             shouldResume: shouldResume
         });
+        if (state.searchQuery && (isValidUrl(state.searchQuery) || isFilePath(state.searchQuery))) {
+            state.descriptionPath = state.searchQuery;
+            return;
+        }
         state.searchResults = await searchCallBack(state.searchQuery);
         if (state.searchResults && Object.keys(state.searchResults).length > 0) {
             return (input: MultiStepInput) => pickSearchResult(input, state);
         } else {
-            state.descriptionPath = state.searchQuery;
-            return (input: MultiStepInput) => inputPathOrUrl(input, state);
-        }
-    }
-
-    async function inputPathOrUrl(input: MultiStepInput, state: Partial<OpenState>) {
```

Comment:
I remove this because it was only referenced in the else statement above, which would set state.descriptionPath = state.searchQuery; then call `inputPathOrUrl` that checks if state.descriptionPath is set, ofcourse it's set because we just set it, and return. The showInputBox would not be ever called and it's also a repeat of the calling function `inputSearch`